src/makeCalculator.js

```diff
@@ -5,6 +5,41 @@
  */
 function makeCalculator() {
   // write code here
+  return {
+    result: 0,
+    add(calculator, number) {
+      calculator.result += number;
+
+      return calculator;
```

Comment:
[NAMING] - The `add` method should not take `calculator` as a parameter. The methods should use `this` to refer to the calculator object itself. The same applies to `subtract`, `multiply`, and `divide` methods. Rewrite the methods to use `this` instead of passing the calculator object.